client/extensions/wp-super-cache/components/advanced/expiry-time.jsx

```diff
@@ -0,0 +1,213 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { pick } from 'lodash';
+import moment from 'moment';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+import FormSelect from 'components/forms/form-select';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import FormTextInput from 'components/forms/form-text-input';
+import FormToggle from 'components/forms/form-toggle/compact';
+import SectionHeader from 'components/section-header';
+import WrapSettingsForm from '../wrap-settings-form';
+
+const ExpiryTime = ( {
+	fields: {
+		cache_gc_email_me,
+		cache_max_time,
+		cache_schedule_interval,
+		cache_schedule_type,
+		cache_scheduled_time,
+		cache_time_interval,
+		wp_cache_next_gc,
+		wp_cache_preload_on,
+	},
+	handleChange,
+	handleRadio,
+	handleSelect,
+	handleToggle,
+	translate,
+} ) => {
+	const renderCacheTimeout = () => {
+		return (
+			<FormFieldset>
+				<FormLabel htmlFor="cache_max_time">
+					{ translate( 'Cache Timeout' ) }
+				</FormLabel>
+
+				<FormTextInput
+					className="wp-super-cache__cache-timeout"
+					onChange={ handleChange( 'cache_max_time' ) }
+					value={ cache_max_time || '' } />
+				{ translate( 'seconds' ) }
+				<FormSettingExplanation>
+					{
+						translate(
+							'How long should cached pages remain fresh? Set to 0 to disable garbage collection. ' +
+							'A good starting point is 3600 seconds.'
+						)
+					}
+				</FormSettingExplanation>
+			</FormFieldset>
+		);
+	};
+
+	const renderScheduler = () => {
+		return (
+			<FormFieldset className="wp-super-cache__scheduler">
+				<FormLabel>
+					{ translate( 'Scheduler' ) }
+				</FormLabel>
+
+				<FormLabel>
+					<FormRadio
+						checked={ 'interval' === cache_schedule_type }
+						name="cache_schedule_type"
+						onChange={ handleRadio }
+						value="interval" />
+					<span>
+						{ translate( 'Timer' ) }
+						<FormTextInput
+							disabled={ 'interval' !== cache_schedule_type }
+							onChange={ handleChange( 'cache_time_interval' ) }
+							value={ cache_time_interval || '' } />
+						{ translate( 'seconds' ) }
+					</span>
+				</FormLabel>
+				<FormSettingExplanation isIndented>
+					{ translate( 'Check for stale cached files every interval seconds.' ) }
+				</FormSettingExplanation>
+
+				<FormLabel>
+					<FormRadio
+						checked={ 'time' === cache_schedule_type }
+						name="cache_schedule_type"
+						onChange={ handleRadio }
+						value="time" />
+					<span>
+						{ translate( 'Clock' ) }
+						<FormTextInput
+							disabled={ 'time' !== cache_schedule_type }
+							onChange={ handleChange( 'cache_scheduled_time' ) }
+							value={ cache_scheduled_time || '' } />
+						{ translate( 'HH:MM' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **33** times:
`translate( 'HH:mm', { context: 'momentjs format string (for LT)'} )` *ES Score: 22*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).